<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jAudio</class>
 <widget class="QMainWindow" name="jAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jAudio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>881</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Record</string>
     </attribute>
     <widget class="QPushButton" name="bRecord">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>271</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/bIcons/iRecord.png</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QGroupBox" name="bActionGroup">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>270</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>254</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/bIcons/iSave.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bPause">
          <property name="text">
           <string>Pause</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/bIcons/iPause.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bUse">
          <property name="text">
           <string>Use</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/bIcons/iUse.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>271</width>
        <height>38</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Device name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lDeviceName">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>--</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>271</width>
        <height>53</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Codecs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lCodecsNumber">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="cCodecs"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>130</y>
        <width>271</width>
        <height>53</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sample rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lSampleNumber">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="cSample"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTextEdit" name="jInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>881</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bRecord</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>bPause</tabstop>
  <tabstop>bUse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
